src/network/socket.tsx

```diff
@@ -33,16 +34,32 @@ export class Socket {
   setKeepAlive = () => {};
   setNoDelay = () => {};
   onData = (data: Uint8Array) => {
-    console.log('lister has not beet setup yet', data);
+    Sentry.addBreadcrumb({
+      category: 'socket',
+      message: `onData lister has not beet setup yet, ${data}`,
+      level: Sentry.Severity.Info,
```

Comment:
are you sure that it shouldn't be error level?

Reply:
I would keep as info